server/php/README.md

```diff
@@ -0,0 +1,24 @@
+### Server
+
+#### Install required packages via Composer
+
+Install required packages with the command:
+
+```shell
+composer install
```

Comment:
Should we add the command here:

```
brew install PHP
brew link php
```

When I install composer, PHP will be installed at the same time. But it is keg-only package. So we still need the `link` command.

Reply:
Added. Thanks @othree